### PR TITLE
openai: fix low timeouts 

### PR DESCRIPTION
**Pull request by theomonnom**
fix ridiculously low timeouts
future work would be to expose those in every client we have

fix https://github.com/livekit/agents/issues/807

### CODE REVIEW

File: livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py [hunk 1]
```diff
@@ -75,11 +75,11 @@ def __init__(
             api_key=api_key,
             base_url=base_url,
             http_client=httpx.AsyncClient(
-                timeout=httpx.Timeout(timeout=30, connect=10, read=5, pool=5),
+                timeout=httpx.Timeout(connect=15.0, read=5.0, write=5.0, pool=5.0),
```
Comment (davidzhao) [inline livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py:78]: the change is from 10s -> 15s for connect? what does removing timeout mean? should we also increase the other timeouts?

**Review by davidzhao — APPROVED** [review]

File: livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py (hunk 1 above)
Comment (theomonnom) [reply livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py:78]: timeout where setting everything to 30.0

Comment (theomonnom) [reply livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py:78]: llm was OK, it's more for TTS/STT that the changes is going from 5 to 15

Comment (davidzhao) [reply livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py:78]: got it, sounds good

**Pull request merged**